gc/base/standard/Scavenger.cpp

```diff
@@ -949,6 +949,12 @@ MM_Scavenger::activateSurvivorCopyScanCache(MM_EnvironmentStandard *env)
 		if ((uintptr_t)cache == MM_AtomicOperations::lockCompareExchange((volatile uintptr_t *)&env->_inactiveSurvivorCopyScanCache, (uintptr_t)cache, (uintptr_t)NULL)) {
 			/* succeded activating */
 			Assert_MM_true(NULL == env->_survivorCopyScanCache);
+			Assert_MM_true(cache->flags & (OMR_SCAVENGER_CACHE_TYPE_CLEARED | OMR_SCAVENGER_CACHE_TYPE_INACTIVE));
```

Comment:
Would you please use explicit `bool` condition in assertion? (here and everywhere is applicable):
```suggestion
			Assert_MM_true(0 != (cache->flags & (OMR_SCAVENGER_CACHE_TYPE_CLEARED | OMR_SCAVENGER_CACHE_TYPE_INACTIVE)));
```